models/video.json

```diff
@@ -0,0 +1,40 @@
+{
+  "title": "video",
+  "type": "object",
+  "description": "Describes a video object that is linked to an event or a place.",
+  "examples": [
+    {
+      "id": "b504cf44-9ab8-4641-9934-38d1cc67242c",
+      "url": "https://www.youtube.com/?v=ewr53d",
+      "copyrightHolder": "publiq"
+    },
+    {
+      "id": "46e9ea9f-fc42-4759-a81b-4308467b7c35",
+      "url": "https://www.vimeo.com/54fdrg"
+    }
+  ],
+  "properties": {
+    "id": {
+      "type": "string",
+      "format": "uuid",
+      "example": "46e9ea9f-fc42-4759-a81b-4308467b7c35",
+      "description": "The id of the added video."
+    },
+    "url": {
+      "type": "string",
+      "format": "url",
+      "example": "https://www.youtube.com/?v=ewr53d",
+      "description": "The unique url of the video. Currently only Youtube and Vimeo sources are allowed."
```

Comment:
Nit: Properties can be documented as only accepting values that match a regex. Would be nice if we could document the fact that only Youtube and Vimeo urls are supported as a regex, so the validation layer in UDB3 also automatically validates that. If that's possible with a regex of course.

Reply:
For example a possible regex could be: `^http(s?)://(www.)?(youtube|vimeo)\.`
This matches any string that starts with `http://` or `https://`, then optionally `www.`, and then either `youtube` or `vimeo` and then a `.` to make sure it's not a URL like `https://youtubefake.com`. (I would not check the domain extension because Youtube and Vimeo support many extensions)

Reply:
Note: Obviously this regex needs to be documented on the property in the POST payload schema, not in this one that's used for the event/place JSON schema.

Reply:
Thanks for doing the heavy lifting and creating the regex 👌